packages/astro/src/core/errors/errors-data.ts

```diff
@@ -454,26 +454,75 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 			`[paginate()] page number param \`${paramName}\` not found in your filepath.`,
 		hint: 'Rename your file to `[page].astro` or `[...page].astro`.',
 	},
+	/**
+	 * @docs
+	 * @see
+	 * - [Assets (Experimental)](https://docs.astro.build/en/guides/assets/)
+	 * - [Image component](https://docs.astro.build/en/guides/assets/#image--astroassets)
+	 * - [Image component#alt](https://docs.astro.build/en/guides/assets/#alt-required)
+	 * @description
+	 * The `alt` property allow users of screen readers and other assistive technologies to understand what your image is supposed to represent. In order to ensure your websites are accessible, the Image component requires that an `alt` be specified.
+	 *
+	 * If the image is merely decorative (i.e. doesn’t contribute to the understanding of the page), set `alt=""` so that screen readers know to ignore the image.
+	 */
 	ImageMissingAlt: {
 		title: 'Missing alt property',
 		code: 3022,
 		message: 'The alt property is required.',
 		hint: "The `alt` property is important for the purpose of accessibility, without it users using screen readers or other assistive technologies won't be able to understand what your image is supposed to represent. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt for more information.",
 	},
+	/**
+	 * @docs
+	 * @see
+	 * - [Image Service API](https://docs.astro.build/en/reference/image-service-reference/)
+	 * @description
+	 * There was an error while loading the configured image service. This can be caused by various factors, such as your image service not properly exporting a compatible object in its default export or the path being wrong.
+	 *
+	 * If you believe that your service is properly configured and this error is wrong, please [open an issue](https://astro.build/issues/)
+	 */
 	InvalidImageService: {
 		title: 'Error while loading image service',
 		code: 3023,
 		message:
 			'There was an error loading the configured image service. Please see the stack trace for more information.',
 	},
+	/**
+	 * @docs
+	 * @message
+	 * Missing width and height attributes for `IMAGE_URL`. When using remote images, both dimensions are always required in order to avoid CLS.
```

Comment:
```suggestion
	 * Missing width and height attributes for `IMAGE_URL`. When using remote images, both dimensions are always required in order to avoid cummulative layout shift (CLS).
```
Just thought I'd add the full term here, for the first time. Then, we can just use CLS. If you prefer to keep THIS message shorter, and add it in the description instead, that's fine too!